core/src/grpc/README.md

```diff
@@ -0,0 +1,2 @@
+We did some modify in gen-milvus, if you run './cpp-gen.sh', generated code will coverage the modify, which may cause some API unavailable.
```

Comment:
We manually add two APIs in "milvus.pd.h":
  api1
  api2
If proto files need be generated again, remember to re-add above APIs.